zc_install/sql/install/mysql_zencart.sql

```diff
@@ -3400,45 +3398,45 @@ INSERT INTO get_terms_to_filter VALUES ('record_company_id', 'TABLE_RECORD_COMPA
 
 INSERT INTO dashboard_widgets (widget_key, widget_group, widget_status, widget_icon, widget_theme, widget_height, widget_width) VALUES
 ('general-statistics', 'general-statistics', 1, 'fa-area-chart', 'bg-light-blue-gradient', 2, 1),
-('order-summary', 'order-statistics', 1, 'fa-shopping-cart', 'bg-light-blue-gradient', 1, 1),
+('order-summary', 'order-summary', 1, 'fa-shopping-cart', 'bg-light-blue-gradient', 1, 1),
 ('new-customers', 'new-customers', 1, 'fa-user-plus', 'bg-light-blue-gradient', 1, 1),
 ('counter-history', 'counter-history', 1, 'fa-calendar', 'bg-light-blue-gradient', 1, 1),
 ('new-orders', 'new-orders', 1, 'fa-shopping-cart', 'bg-light-blue-gradient', 1, 1),
 ('logs', 'logs', 1, 'fa-thumbs-o-up', 'bg-light-blue-gradient', 1, 1)
 ;
 
-INSERT INTO dashboard_widgets_description (widget_key, widget_name, widget_description, language_id) VALUES
-('general-statistics', 'GENERAL_STATISTICS', '', 1),
-('order-summary', 'ORDER_SUMMARY', '', 1),
-('new-customers', 'NEW_CUSTOMERS', '', 1),
-('counter-history', 'COUNTER_HISTORY', '', 1),
-('new-orders', 'NEW_ORDERS', '', 1),
-('logs', 'LOGS', '', 1)
+INSERT INTO dashboard_widgets_description (widget_key, widget_name) VALUES
+('general-statistics', 'GENERAL_STATISTICS'),
+('order-summary', 'ORDER_SUMMARY'),
+('new-customers', 'NEW_CUSTOMERS'),
+('counter-history', 'COUNTER_HISTORY'),
+('new-orders', 'NEW_ORDERS'),
+('logs', 'LOGS')
```

Comment:
This PR looks good. I'm torn as to whether my suggestion to drop this extra table should be part of this PR or separate on its own.  
In the end either is fine.